usagov/usagov-2021
# USAGOV-1440 sayt static and ids

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1440

## Description

The static site wasn't having the drop down menu work on pages other than the "A" page. 

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- You'll have to generate a static site. I suggest using script "scripts/tome-static.sh http://127.0.0.1/" since the CMS form seems to give errors. 
- Once the static site generates go to http://127.0.0.1/agency-index and select any letters other than A. Test that the drop down shows and works. 
- If it does once you merge and deploy to dev please do another check on the dev static site that it is still working as expected. Thank you! 

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
